src/main/features/claim/views/check-and-send.njk

```diff
@@ -2,7 +2,9 @@
 
 {% from "table.njk" import tableStart, tableEnd, row %}
 {% from "form.njk" import csrfProtection, submitButton, errorSummary %}
-{% from "../../../views/macro/statementOfTruth.njk" import statementOfTruth %}
+{% from "statementOfTruth.njk" import statementOfTruth %}
```

Comment:
I think this is using the wrong statementOfTruth macro, I have a feeling that rafal copied it into the app at that location